templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+	<h1>Delete Manufacturer</h1>
+	<p>Are you sure you want to delete manufacturer {{ manufacturer }}</p>
+	<p><i>All Cars with this manufacturer will be also deleted!</i></p>
+	<form action="" method="post">
```

Comment:
The form's `action` attribute is empty, which means it will submit to the current URL. Ensure that this behavior is intended and that the view handling this form submission is correctly set up to process the deletion.